src/types/local_time.rs

```diff
@@ -0,0 +1,93 @@
+#[warn(unused_imports)]
+use napi::{bindgen_prelude::BigInt, Error, Status};
+use scylla::frame::value::CqlTime;
+
+const NANO_SEC_IN_SEC: i64 = 1000000000;
+
+#[napi]
+pub struct LocalTime {
+    pub value: BigInt,
+    pub hour: i64,
+    pub minute: i64,
+    pub second: i64,
+    pub nanosecond: i64,
+    ns_value: i64,
+}
+
+#[napi]
+impl LocalTime {
+    fn convert_to_object(ns_value: i64) -> Self {
+        let mut second: i64 = ns_value / NANO_SEC_IN_SEC;
+        let nanosecond: i64 = ns_value - (second * NANO_SEC_IN_SEC);
+
+        let mut minute = second / 60;
+        let hour = minute / 60;
+
+        second %= 60;
+        minute %= 60;
+
+        LocalTime {
+            value: BigInt::from(ns_value),
+            hour,
+            minute,
+            second,
+            nanosecond,
+            ns_value,
+        }
+    }
+
+    #[napi]
+    pub fn new(total_nanoseconds: BigInt) -> napi::Result<Self> {
+        let (ns_value, is_lossless) = total_nanoseconds.get_i64();
+
+        if !is_lossless {
+            return Err(Error::new(
+                Status::GenericFailure,
+                "Nanoseconds must not overflow i64",
+            ));
+        }
+
+        Ok(Self::convert_to_object(ns_value))
+    }
+
+    //format: hh:MM:ss.ns
+    #[napi]
+    pub fn to_format(&self) -> String {
```

Comment:
Why the function is called to_format, to_string would not be better?

Reply:
When name of function is `to_string`, rust wants implement trait `Display`...